### ISSUE TITLE
Webpack 5 DeprecationWarning

### ISSUE THREAD

**Issue by dotnetshadow** [zoobestik/csso-webpack-plugin#24]
I get the following warning when using the plugin:

`(node:17184) [DEP_WEBPACK_COMPILATION_OPTIMIZE_CHUNK_ASSETS] DeprecationWarning: optimizeChunkAssets is deprecated (use Compilation.hook.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)`

Everything works fine, but the warning is shown

### Specifications
- **CSSO Plugin version**: v2.0.0-beta.1
- **Webpack version**: v5.24.0

**/cc** @zoobestik

**Comment by in-in** [comment]
I have two warnings from csso-webpack-plugin
```sh
(node:45418) [DEP_WEBPACK_COMPILATION_OPTIMIZE_CHUNK_ASSETS] DeprecationWarning: optimizeChunkAssets is deprecated (use Compilation.hooks.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:45418) [DEP_WEBPACK_DEPRECATION_ARRAY_TO_SET] DeprecationWarning: Compilation.chunks was changed from Array to Set (using Array method 'map' is deprecated)
```

### Specifications
- **CSSO Plugin version**: v2.0.0-beta.1
- **Webpack version**: v5.24.1
- **Node + npm versions**: v14.15.4 + 7.5.4

**Comment by zoobestik** [comment]
**Fixed in [2.0.0-beta.2](https://github.com/zoobestik/csso-webpack-plugin/releases/tag/v2.0.0-beta.2).**
Let me know if you will have problems.
